pop-cli-for-appchains/guides/launch-a-chain/deploy-a-chain-external-provider.md

```diff
@@ -0,0 +1,80 @@
+---
+description:
+  This guide explains how to deploy a chain using an external provider.
+---
+
+# Deploy a Chain Using an External Provider
+
+Pop CLI integrates an external provider for chain deployment, streamlining the process with seamless automation. It handles registration, as detailed in [Launch a Chain to Paseo](./launch-a-chain-to-paseo.md), and once the chain is registered, Pop CLI completes the deployment by integrating with the provider. The provider manages collators and offers a portal UI for monitoring your chain’s status.
+
+Currently, the only supported deployment provider is the [Polkadot Development Portal](https://www.deploypolkadot.xyz/), which supports a limited set of templates.
+
+Execute the following command to start the interactive deployment process:
+```shell
+pop up
+```
+And follow the interactive guide 
+
+<figure><img src="../.gitbook/assets/pdpflow.png" alt="pop up"><figcaption><p>pop up</p></figcaption></figure>
```

Comment:
The GIF was too large for GitHub, and lowering its quality made it unusable. An image should be enough.